docs/site/Inside-Loopback-Application.md

```diff
@@ -0,0 +1,173 @@
+---
+lang: en
+title: 'Inside a LoopBack Application'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Inside-LoopBack-Application.html
+---
+
+## APIs in a digital era
+
+Digital transformation is changing how data and services are shared across Web/Mobile/IoT platforms.
+APIs allow easy and secure access to digital resources and capabilities of an organization by internal and enternal parties.
+
+Developers are looking out for more productive ways of creating and consuming APIs.
+
+Consider a typical `Online Shopping` site. First the UI requirements are gathered. Customers would be provided a home page, a product list, a shopping cart and an order history page.
+
+![Shopping Application](imgs/shopping-app.png)
+
+Next, the API requirements are assessed. If the capability to choose products, add to cart and submit an order are exposed as secure APIs, they can be used by an in-house web application as well as a third party mobile app developer.
+
+Once the API requirements are assessed,
+
+- Front end designers would want to try the `Online Shopping` UI experience.
```

Comment:
I'm thinking to put the focus back to how LB can help.  i.e. 

LoopBack can be used to satisfy the requirements from different developers:
- Frontend Developers: ...
- API Developers: ...
- Backend Developers: ...

Reply:
I see.. you want to make LoopBack the main topic and break down from there to how developers can use it. let me see how that looks.